apps/web/src/app/api/convert-video/route.ts

```
@@ -0,0 +1,144 @@
+
+function sanitizeContentDispositionName({
+	rawName,
+}: {
+	rawName: string;
+}): string {
+	// Strip characters that can break the Content-Disposition header (quotes,
+	// backslashes, CR/LF, control bytes) and replace non-ASCII so the
+	// "filename=" parameter stays unambiguous across clients.
+	const stripped = rawName
+		// eslint-disable-next-line no-control-regex
+		.replace(/[\\"\r\n\x00-\x1f]/g, "_")
```

Biome (2.4.15)

[error] Unexpected control character in a regular expression.

(lint/suspicious/noControlCharactersInRegex)